badger_db.go

```diff
@@ -28,7 +28,12 @@ func NewBadgerDB(dbName, dir string) (*BadgerDB, error) {
 	if err := os.MkdirAll(path, 0755); err != nil {
 		return nil, err
 	}
+
+
 	opts := badger.DefaultOptions(path)
+	opts.IndexCacheSize = 256 << 20 // 100 mb or some other size based on the amount of data
+	opts.DetectConflicts = false
+	opts.NumGoroutines = 64
```

Comment:
@creachadair this was a guess.

Supposing that we're rejecting any test that doesn't occur against a live node, do you have any thoughts on getting data behind this?

Reply:
It doesn't necessarily have to be a live node, I just think we should have some reason for choosing a value besides a general feeling 🙂 

Even a load test could be helpful, or a testnet that gets enough load for the difference to matter.